app_dart/lib/src/request_handlers/github/webhook_subscription.dart

```diff
@@ -138,6 +145,10 @@ class GithubWebhookSubscription extends SubscriptionHandler {
       case 'synchronize':
         // This indicates the PR has new commits. We need to cancel old jobs
         // and schedule new ones.
+        await scheduler.cancelPreSubmitTargets(
```

Comment:
Can you add a test to verify this?

Reply:
Reverted this call back to the triggerPresubmitBuilds method.